cscs-checks/apps/amber/amber.py

```diff
@@ -57,103 +54,68 @@
     },
 }
 
+REFERENCE_ENERGY = {
+    # every system has a different reference energy and drift
+    'Cellulose_production_NVE': (-443246.0, 5.0E-05),
+    'FactorIX_production_NVE': (-234188.0, 1.0E-04),
+    'JAC_production_NVE_4fs': (-44810.0, 1.0E-03),
+    'JAC_production_NVE': (-58138.0, 5.0E-04),
+}
 
-class AmberBaseCheck(rfm.RunOnlyRegressionTest):
-    valid_prog_environs = ['builtin']
-    strict_check = False
-    modules = ['Amber']
-    extra_resources = {
-        'switches': {
-            'num_switches': 1
-        }
-    }
-    maintainers = ['VH', 'SO']
-    tags = {'scs', 'external-resources'}
 
+class AmberCheck(AmberBaseCheck):
     benchmark = parameter([
         # NVE simulations
         'Cellulose_production_NVE',
         'FactorIX_production_NVE',
         'JAC_production_NVE_4fs',
         'JAC_production_NVE',
     ])
+    strict_check = False
+    extra_resources = {
+        'switches': {
+            'num_switches': 1
+        }
+    }
+    ener_ref = REFERENCE_ENERGY
+    output_file = 'amber.out'
+    maintainers = ['VH', 'SO']
 
     @run_after('init')
-    def download_files(self):
+    def source_install(self):
```

Comment:
I think you're not installing anything here. These are just files for the checks themselves. So probably the previous name made more sense.